tests/e2e/fixtures/storage_efs_dependencies.py

```diff
@@ -55,6 +55,17 @@ def callback():
 
     wait_for(callback)
 
+def wait_on_mount_target_status(desired_status, efs_client, file_system_id):
+    def callback():
+        response = efs_client.describe_file_systems(
+            FileSystemId=file_system_id,
+        )
+        number_of_mount_targets = response["FileSystems"][0]["NumberOfMountTargets"]
+        print(f"{file_system_id} has {number_of_mount_targets} mount targets .... waiting")
+        if desired_status == "deleted": assert number_of_mount_targets == 0
+        else: assert number_of_mount_targets > 0
```

Comment:
is it also necessary to verify desired_status == actual_status?

Reply:
the deleted mount target does not have a "status" field. I am just using that string to mean that I should be checking for 0 mount targets attached.